pgmpy/factors/continuous/distributions/CustomDistribution.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from scipy import integrate
 
-from pgmpy.factors.distributions import BaseDistribution
+from pgmpy.factors import BaseDistribution
 
 
```

Comment:
I would think that each distribution, such as dirichlet or beta could be added into the repo itself instead of custom distribution class as they would be useful to all given that they are conjugate priors that are always used. thoughts @ankurankan ?